### PR TITLE
DOCS-3593 [Trace Explorer][2] Facets article

### PR DESCRIPTION
**Pull request by dussault-antoine**
### What does this PR do?

Second iteration of the Trace Explorer docs revamp
**Note:** Some links are not working, because some article are not yet created. These articles are coming in next PRs

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/trace_explorer/visualize.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+The list view displays a list of spans that match the selected context, defined by the [search bar query][1] filter and a [time range][2].
+
+In the **Show** row, choose which information of interest to display as columns. Manage the columns by either:
+
+- interacting with the table header row to **sort**, **rearrange**, or **remove** columns.
+- selecting a facet from the facet panel on the left, or from the trace side panel after clicking on a specific span, to **add** a column for a field. You can also add columns from with the **Options** button.
+
+{{< img src="tracing/trace_explorer/visualize/list_view_table_controls.mp4" alt="Configuring the display table" video=true style="width:80%;">}}
+
+{{< site-region region="us,eu,us3,us5,gov" >}}
```
Comment (kayayarai) [inline content/en/tracing/trace_explorer/visualize.md:26]: Is there a site that should be removed from this list? Or can we remove the site-region section tags?

File: content/en/tracing/trace_explorer/visualize.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+## Overview
+
+Visualizations define how the queried span data is displayed. Select relevant visualizations to surface valuable information, such as a **list** for individual events, or as **timeseries** or **top lists** for aggregates. 
+
+## List view
+
+The list view displays a list of spans that match the selected context, defined by the [search bar query][1] filter and a [time range][2].
+
+In the **Show** row, choose which information of interest to display as columns. Manage the columns by either:
```
Comment (kayayarai) [inline content/en/tracing/trace_explorer/visualize.md:19]: Previously, this said "In the choose which", so please make sure that I added an appropriate wording. :)

**Review by kayayarai — CHANGES_REQUESTED** [review]
I made quite a few editorial updates. A couple questions for you, but otherwise looks good. I removed the future links (otherwise it turns our entire build RED until they're fixed). We can add them back in the PRs where you add those files.

File: content/en/tracing/trace_explorer/visualize.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+The following Trace Explorer timeseries view shows the evolution of the top ten resource names of the service `shopist-web-ui` according to the 95th percentile of `Duration` over the past four hours:
+
+{{< img src="tracing/trace_explorer/visualize/timeseries_view.png" alt="Timeseries view" style="width:100%;">}}
+
+Choose additional display options for timeseries: the **roll-up interval**, whether you **display** results as **bars** (recommended for counts and unique counts), **lines** (recommended for statistical aggregations) or **areas**, and the **colorset**.
+
+## Top list
+
+Use a top list to visualize a span count, a count of unique tag values, or a measure split by one tag dimension.
```
Comment (kayayarai) [inline content/en/tracing/trace_explorer/visualize.md:69]: is "a count of unique tag values" correct? The previous wording didn't make sense to me. If this is still off, let's workshop what it should say.

File: content/en/tracing/trace_explorer/visualize.md (hunk 2 above)
Comment (dussault-antoine) [inline content/en/tracing/trace_explorer/visualize.md:19]:
```suggestion
In the table, choose which information of interest to display as columns. Manage the columns by either:
```

File: content/en/tracing/trace_explorer/visualize.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+- interacting with the table header row to **sort**, **rearrange**, or **remove** columns.
+- selecting a facet from the facet panel on the left, or from the trace side panel after clicking on a specific span, to **add** a column for a field. You can also add columns from with the **Options** button.
+
+{{< img src="tracing/trace_explorer/visualize/list_view_table_controls.mp4" alt="Configuring the display table" video=true style="width:80%;">}}
+
+{{< site-region region="us,eu,us3,us5,gov" >}}
+The default sort for spans in the list visualization is by timestamp, with the most recent spans on top. To surface spans with lowest or highest value for a measure first, or to sort your spans lexicographically for the value of a tag, specify that column as the **by** column.
+
+{{< /site-region >}}
```
Comment (dussault-antoine) [inline content/en/tracing/trace_explorer/visualize.md:29]:
```suggestion
The default sort for spans in the list visualization is by timestamp, with the most recent spans on top. To surface spans with lowest or highest value for a measure first, or to sort your spans lexicographically for the value of a tag, specify that column as the **by** column.

```

File: content/en/tracing/trace_explorer/visualize.md (hunk 3 above)
Comment (dussault-antoine) [reply content/en/tracing/trace_explorer/visualize.md:69]: This makes sense, thanks for modifying

**Review by kayayarai — APPROVED** [review]

**Pull request merged**